### DISCUSSION TITLE
Starting Applications thorugh explorer.exe

### DISCUSSION THREAD

**Discussion by Cyb3r-Monk** [cmu-sei/GHOSTS#106] [Ideas]
Hi, 

I just started using the tool and loved it!

Something that I see is that Office applications are launched through svchost.exe(DCOM service ->  "svchost.exe -k DcomLaunch -p"). Also, Chrome is launched through chromedriver.exe I was wondering if it was possible to launch the applications through explorer.exe like a normal user. 
<img width="815" alt="image" src="https://user-images.githubusercontent.com/26627224/184536422-cc0d8827-363f-4aeb-a7ae-109b1f3c0c0f.png">

**Comment by sei-dupdyke** [comment]
Hi, thanks for checking out the project.

Unfortunately, I don't know of a way to automate Office or the browser without using specific automation libraries to do so. For our uses, this is fine, since most of the activity is observed via the network or logs, where the chain that you show above is sort of obscured. I'm always looking for new ways to automate applications, however, so if you know of some method that removes the problem you highlight, please let me know.

**Reply by Cyb3r-Monk** [reply]
Thanks for the answer, Dustin! 

I think it's only possible via process injection which is not feasible. If I come across a method, I'll let you know. 
Btw, the screenshot is an excerpt from process creation logs.
